src/gl/interface/gl-objects-buffers.ads

```diff
@@ -13,10 +14,14 @@ package GL.Objects.Buffers is
                          Dynamic_Draw, Dynamic_Read, Dynamic_Copy);
 
    type Buffer_Target (<>) is tagged limited private;
+   type Texture_Buffer_Target (<>) is tagged limited private;
```

Comment:
Inherit from `Buffer_Target` and move this type below all `Buffer_Target` subroutines to inherit the subroutines of `Buffer_Target`. That will get you rid of the duplicates.

Reply:
Have done (hopefully correctly).
Duplicates deleted